lib/y-richtext.coffee

```diff
@@ -172,12 +172,20 @@ class YRichText extends BaseClass
 
       if delta.insert?
         insertHelper thisObj, position, delta.insert
-        from = thisObj._model.getContent("characters").ref(position)
-        to = thisObj._model.getContent("characters").ref(position+delta.insert.length)
-        thisObj._model.getContent("selections").select from, to, delta_selections
-        thisObj._model.getContent("selections").unselect from, to, delta_unselections
-
-        return position + delta.insert.length
+        length =
```

Comment:
That's true that `delta.insert` has to be defined, but it doesn't prevent it from being an object or a function, in which case the behavior will be bad.
I'm adding a test to fail in case it's neither a string or a number, and comment the section.